### ISSUE TITLE
Unknown function: netrw#LocalBrowseCheck when trying to open file

### ISSUE THREAD

**Issue by skipadu** [nvim-telescope/telescope.nvim#1565]
### Description

When using telescope picker, `find_files`, and when trying to open a file an error is shown and no file is opened:
```
E5108: Error executing lua ...im/plugged/telescope.nvim/lua/telescope/actions/init.lua:251: Vim(call):E117: Unknown function: netrw#LocalBrowseCheck
```
I'm using git worktrees. I think this is related to worktrees as it does not seem to have this kind of problem when in another project that is not using git worktrees.

_For easy use of worktrees in nvim, I use [git-worktree.nvim](https://github.com/ThePrimeagen/git-worktree.nvim)_

### Neovim version

```markdown
NVIM v0.6.0
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macOS 12.0.1

### checkhealth telescope

```markdown
telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
    - OK: plenary installed.
    - OK: nvim-treesitter installed.
## Checking external dependencies
    - OK: rg: found ripgrep 13.0.0
    - OK: fd: found fd 8.2.1
## ===== Installed extensions =====
## Telescope Extension: `git_worktree`
   - INFO: No healthcheck provided
```

### Steps to reproduce

1.  Open project that uses git worktrees
2. Go to wanted worktree
3. `:Telescope find_files`
4. Select any file and press Enter/Return
5. Error is shown

### Expected behavior

Expecting to have file opened when selecting it from the picker and not seeing the error.

### Actual behavior

Selecting the file from picker and pressing Enter/Return and error is show and file is not opened.

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'
local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      {
        'nvim-telescope/telescope.nvim',
        requires = {
          'nvim-lua/plenary.nvim',
          { 'nvim-telescope/telescope-fzf-native.nvim', run = 'make' },
        },
      },
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
      'ThePrimeagen/git-worktree.nvim',
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
      display = { non_interactive = true },
    },
  }
end
_G.load_config = function()
  require('telescope').setup()
  require('telescope').load_extension('fzf')
  -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end
if vim.fn.isdirectory(install_path) == 0 then
  print("Installing Telescope and dependencies.")
  vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end
load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]
```

**Comment by Conni2461** [comment]
no idea whats happening here. But we dont have a single `netrw#` function call. Did you reproduce this with with the minimal rc you've provided? running `nvim -nu minrc.lua`?

**Comment by skipadu** [comment]
**TLDR; Issue solved: error in my config**

Couldn't reproduce the error with the minimal rc when I tested it now. 

And I think I have found the culprit now:

I had added long time ago and had totally forgotten it, while I still used NerdTree plugin this line to my config to prevent the netrw for showing up all the time:
```vim
let loaded_netrw = 0
```

Haven't had any issues with that until I got those errors when using telescope with git worktrees.

It seems to fix the issue when changed this netrw config to (fingers crossed):

```vim
let loaded_netrwPlugin = 1
```











Following paste is the whole error that confused me and I thought it was because of Telescope. Putting it here if someone else might have similar error with their config.

```
E5108: Error executing lua ...im/plugged/telescope.nvim/lua/telescope/actions/init.lua:251: Vim(call):E117: Unknown function: netrw#LocalBrowseCheck
stack traceback:
        [C]: in function 'nvim_win_close'
        ...im/plugged/telescope.nvim/lua/telescope/actions/init.lua:251: in function 'run_replace_or_original'
        ...nvim/plugged/telescope.nvim/lua/telescope/actions/mt.lua:30: in function '_close'
        ...im/plugged/telescope.nvim/lua/telescope/actions/init.lua:257: in function 'run_replace_or_original'
        ...nvim/plugged/telescope.nvim/lua/telescope/actions/mt.lua:30: in function 'close'
        ...vim/plugged/telescope.nvim/lua/telescope/actions/set.lua:125: in function 'run_replace_or_original'
        ...nvim/plugged/telescope.nvim/lua/telescope/actions/mt.lua:30: in function 'run_replace_or_original'
        ...nvim/plugged/telescope.nvim/lua/telescope/actions/mt.lua:30: in function 'run_replace_or_original'
        ...nvim/plugged/telescope.nvim/lua/telescope/actions/mt.lua:30: in function 'key_func'
        ...e/nvim/plugged/telescope.nvim/lua/telescope/mappings.lua:235: in function 'execute_keymap'
        [string ":lua"]:1: in main chunk

```


Sorry for the hassle and the trouble I caused with this.
